iBurn/src/main/java/com/gaiagps/iburn/api/IBurnService.java

```diff
@@ -323,8 +324,10 @@ private Single<Long> updateEvents(DataProvider provider) {
         //final SimpleDateFormat mahineDateFormatter = new SimpleDateFormat("yyyy-MM-dd HH:mm:ssX", Locale.US);
         // Date format for human-readable specific-time
         final SimpleDateFormat timeDayFormatter = new SimpleDateFormat("EE M/d h:mm a", Locale.US);
+        timeDayFormatter.setTimeZone(DateUtil.PLAYA_TIME_ZONE);
         // Date format for human-readable all-day
         final SimpleDateFormat dayFormatter = new SimpleDateFormat("EE M/d", Locale.US);
+        timeDayFormatter.setTimeZone(DateUtil.PLAYA_TIME_ZONE);
```

Comment:
I think there's a typo here 

```suggestion
        dayFormatter.setTimeZone(DateUtil.PLAYA_TIME_ZONE);
```